lib/ferver/found_file.rb

```diff
@@ -1,12 +1,16 @@
 module Ferver
   class FoundFile
-    attr_reader :file_name, :path_to_file
+    require_relative 'hash_cache'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.